oak-doc/src/site/markdown/differences.md

```diff
@@ -316,6 +318,19 @@ Node Name Length Limit
 
 With the document storage backend (MongoDB, RDBMS), there is currently 
 a limit of 150 UTF-8 bytes on the length of the node names.
-See also OAK-2644.
+See also [OAK-2644](https://issues.apache.org/jira/browse/OAK-2644).
+
+Session Attributes
+------------------
+
+Oak exposes the following attributes via [`Session.getAttribute(...)`][1] and [`Session.getAttributeNames()`][2].
```

Comment:
that's not entirely correct.... it also exposes credentials attributes as intended by the JCR specification. maybe rather state something like: in addition to attributes defined on credentials.......

Reply:
Fixed in https://github.com/apache/jackrabbit-oak/pull/291/commits/3d7a7e57c2ca1a117acc0b65c45c2f7739a966e1.